adapters/consumable/consumable.go

```diff
@@ -139,31 +145,45 @@ func (a *ConsumableAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *
 	gdpr := bidGdpr{}
 
 	ccpaPolicy, err := ccpa.ReadFromRequest(request)
-	if err == nil {
-		body.CCPA = ccpaPolicy.Consent
+	if err != nil {
+		errs = append(errs, err)
 	}
 
+	body.CCPA = ccpaPolicy.Consent
+
 	// TODO: Replace with gdpr.ReadPolicy when it is available
 	if request.Regs != nil && request.Regs.Ext != nil {
 		var extRegs openrtb_ext.ExtRegs
-		if err := json.Unmarshal(request.Regs.Ext, &extRegs); err == nil {
-			if extRegs.GDPR != nil {
-				applies := *extRegs.GDPR != 0
-				gdpr.Applies = &applies
-				body.GDPR = &gdpr
-			}
+		if err := json.Unmarshal(request.Regs.Ext, &extRegs); err != nil {
+			errs = append(errs, err)
+		}
+		if extRegs.GDPR != nil {
+			applies := *extRegs.GDPR != 0
+			gdpr.Applies = &applies
+			body.GDPR = &gdpr
 		}
 	}
 
 	// TODO: Replace with gdpr.ReadPolicy when it is available
```

Comment:
I know this may not be in the scope of this PR, but is the comment still needed?